### ISSUE TITLE
Allow router to route to other ports than 80 and 443

### ISSUE THREAD

**Issue by reneweteling** [flynn/flynn#4216]
Hi Guys,

It would be awesome if we could change give routes explicit incoming ports. So, for instance, we have the Flynn dashboard. Which is awesome, but also really freaky dangerous. 

The same goes for a rails app that I've made to automatically add SSL certs to Flynn routes, and periodically backups resources (Postgres / mongo etc), also something that should not be on the big bad WWW. (when its done ill publish the app so that more peeps can share the love)

Id likes to add the dashboard to a weird port, block it on my firewall and when I want to access it, just create an ssh tunnel and view the dashboard over my ssh tunnel. ( like Rancher does )

Thanks guys!

**Comment by titanous** [comment]
This was implemented in #4000.

**Comment by reneweteling** [comment]
Hi @titanous thaks so much for this great product. Only im a bit stuck

```bash
➜  flynn-backup git:(master) flynn -a dashboard route add http -p 3010 dashboard.weteling.com
validation_error: Port not reserved for HTTP traffic
```

i do get this working when i choose TCP, only i need HTTP. Do you have a clueue what can be wrong here?

Thanks so much!

p.s. Sorry about posting in a closed issue
